client/components/backup-storage-space/hooks.tsx

```diff
@@ -111,3 +111,50 @@ export const useDaysOfBackupsSavedText = (
 		);
 	}, [ translate, daysOfBackupsSaved, siteSlug ] );
 };
+
+/**
+ * The idea is to convert any storage amount in bytes to a human readable format.
+ *
+ * @param storageInBytes The storage amount in bytes
+ * @returns				 The storage amount in a human readable format
+ */
+export const useStorageText = ( storageInBytes: number ): TranslateResult | string => {
+	const translate = useTranslate();
+
+	return useMemo( () => {
```

Comment:
Is the memoization here worth the effort? What's the performance impact with and without it?